presidio-analyzer/setup.py

```diff
@@ -11,8 +11,11 @@
 with open(path.join(this_directory, 'README.MD'), encoding='utf-8') as f:
     long_description = f.read()
 
-with open(os.path.join(parent_directory, "VERSION")) as version_file:
-    __version__ = version_file.read().strip()
+try:
+    with open(os.path.join(parent_directory, "VERSION")) as version_file:
+        __version__ = version_file.read().strip()
+except:  # noqa: E722
+    __version__ = os.environ.get("VERSION", "0.0.1-alpha")
```

Comment:
is this to avoid the image-redactor current version? what if the version will change?

Reply:
This is called by requirements.txt of the e2e test project, and VERSION file does not exists in this context. For the rest of the projects, the VERSION file will exists and the value will be taken from there.